src/main/java/com/mjd/bank/services/impl/AccountServiceImpl.java

```diff
@@ -35,4 +45,21 @@ public SimpleMessageResponse deposit(Long ownerId, Long accountId, BigDecimal am
 
     return new SimpleMessageResponse(String.format("Deposit of %s to account %s completed", amount, accountId));
   }
+
+  @Override
+  public SimpleMessageResponse create(Long ownerId, CreationRequest creationRequest) {
+
+    AppUser owner = appUserRepository.findById(ownerId)
+            .orElseThrow(() ->new NotFoundException("The user with ID " + ownerId + " doesn't exist"));
+
+    transactionsUtils.isAccountOwner(ownerId, creationRequest.getOwnerId());
+
+    if (AccountType.getAccountType(creationRequest.getType().toUpperCase())==AccountType.UNKNOWN){
+        throw new IncorrectAccountTypeException("The account type doesn't exist");
+    }else {
+        Account account = new Account(AccountType.getAccountType(creationRequest.getType().toUpperCase()),owner);
+        accountRepository.save(account);
+        return new SimpleMessageResponse("Account created successfully");
+      }
```

Comment:
El else es innecesario
```suggestion
    }
    
    Account account = new Account(AccountType.getAccountType(creationRequest.getType().toUpperCase()),owner);
        accountRepository.save(account);
    return new SimpleMessageResponse("Account created successfully");
}
```